torchtune/datasets/_text_completion.py

```diff
@@ -94,6 +97,7 @@ def text_completion_dataset(
         ...   column="text",
         ...   max_seq_len=2096,
         ...   data_dir="realnewslike",
+        ...   packed=True,
```

Comment:
For this example, let's actually set packed=False because for this dataset most hardware setups will OOM if packed